### PR TITLE
Server - health check

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
if you hit up /healthcheck it will return something like

```
{"dbConnected":true,"dbDataLoad":true,"uptime":226}
```

I havn't found a way to make sqlite fail because linux is smarter than I am. Sequelize health check method was taken from their official documentation. http://docs.sequelizejs.com/manual/installation/getting-started.html#test-the-connection

#79 Tick

### CODE REVIEW

**Comment by sdunster** [comment]
Awesome but to close #79 it also has to fail if the test data is not yet initialised. This way we can delay the integration tests until we know the server is ready.

File: server/src/main.js [hunk 1]
```diff
@@ -81,6 +81,29 @@ app.use(
...
+  let status = 500;
+  const response = {};
+  response.uptime = Math.round(process.uptime());
+  try {
+    db
+      .authenticate()
+      .then(() => {
+        status = 200;
+        response.dbConnector = 'healthy';
+        res.status(status).json(response);
```
Comment (sdunster) [inline server/src/main.js]: i think you can probably just omit the status part of this - just let express return what it thinks is sane.

Comment (OSPFNeighbour) [reply server/src/main.js]: run the risk of a unhealthy result returning a code 200 if you do that.

Comment (OSPFNeighbour) [reply server/src/main.js]: (cause we would want a 500 or something if there is a problem)

**Comment by OSPFNeighbour** [comment]
Technically #79 is only the server side and #80 is the test side.

**Pull request merged**